components/automate-deployment/pkg/history/event.go

```diff
@@ -0,0 +1,34 @@
+package history
+
+import "time"
+
+type EventID string
```

Comment:
Why a string and not a `uuid` type directly?

Reply:
i hadn't decided i was going to use uuid when i wrote this line